src/vue/pages/Fees.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div class="fee" v-if="fees">
+    <!--
+      :key is a hack to ensure that the component will be updated
+      after computed calculated
+      -->
+    <select-field-custom
+      v-model="filters.asset"
+      :values="assetCodes"
+      :key="filters.asset"
+    />
+    <div class="fee__asset-info">
```

Comment:
let's remove the `fee__asset-info` `div`, add the wrapper to select and apply the `margin-bottom` prop to it

Reply:
Resolved, updated